curriculum/challenges/english/25-front-end-development/workshop-cafe-menu/5f3c866dbf362f99b9a0c6d0.md

```diff
@@ -21,21 +21,21 @@ You should use the `.item p` selector.
 
 ```js
 const hasItemP = new __helpers.CSSHelp(document).getStyle('.item p');
-assert(hasItemP);
+assert.exists(hasItemP);
 ```
 
 You should set the `display` property to `inline-block`.
 
 ```js
 const hasDisplay = new __helpers.CSSHelp(document).getCSSRules().some(x => x.style.display === 'inline-block');
-assert(hasDisplay);
+assert.exists(hasDisplay);
```

Comment:
you should check what the return value of `some` is before deciding what is the best assert method